common/src/main/cpp/extensions/openxr_fb_render_model.cpp

```diff
@@ -0,0 +1,131 @@
+/**************************************************************************/
+/*  openxr_fb_render_model.cpp                                            */
```

Comment:
Can you move this implementation into a `classes` directory: `common/src/main/cpp/classes`. 

This will enable us to distinguish between extension wrappers implementations in the `extensions` directory and nodes / classes used to expose the functionality to end users.